Http.cpp

```diff
@@ -48,7 +48,9 @@
 #define REGISTRATION_STATUS "AT+CREG?\r\n"
 #define SIGNAL_QUALITY "AT+CSQ\r\n"
 #define READ_VOLTAGE "AT+CBC\r\n"
-#define SLEEP_MODE "AT+CSCLK=1\r\n"
+#define SLEEP_MODE "AT+CSCLK=2\r\n"
```

Comment:
I guess by using this sleep mode the first time the user call the method `sleep` the module will enter in sleep mode always after 5s of inactivity henceforth.
I'd modify [this](https://github.com/carrascoacd/ArduinoSIM800L/blob/master/Http.cpp#L171) method in order to send a `AT+CSCLK=0` in order to disable it.
What do you think?

Reply:
Yes it would make sense to send `AT+CSCLK=0` after AT command in order to wake up the module. The reason I changed from mode 1 to two is because first one didn't work for me. Probably because I didn't have **reset** pin connected.